fuel-core/Cargo.toml

```diff
@@ -28,15 +28,15 @@ clap = { version = "3.1", features = ["env", "derive"] }
 derive_more = { version = "0.99" }
 dirs = "3.0"
 env_logger = "0.9"
-fuel-asm = { version = "0.2", features = ["serde-types"] }
+fuel-asm = { version = "0.3", features = ["serde-types"] }
 fuel-core-interfaces = { path = "../fuel-core-interfaces", version = "0.4.2" }
 fuel-crypto = { version = "0.3" }
```

Comment:
While you're here, should you update `fuel-crypto` to `0.4`?

Reply:
unfortunately, looks like we'd have to update all the upstream deps first (fuel-tx and fuel-vm). We should probably make it so fuel-core just consumes whatever version of fuel-crypto is re-exported by the VM instead of managing the version here ourselves.

Reply:
we'd have to update fuel-vm to make that re-export possible first though.

Reply:
Made an issue to capture this here https://github.com/FuelLabs/fuel-core/issues/236